app/services/manage_braintree_donation.rb

```diff
@@ -34,21 +34,22 @@ def queue_message
   def organize_params
     {
         donationpage: {
-            name: "#{page.slug}-donation",
-            payment_account: 'Default Import Stub'
+            name:             "#{page.slug}-donation",
+            payment_account:  'Default Import Stub'
         },
         order: {
-            amount: transaction.amount,
-            card_num: card_num,
-            card_code: '007',
+            amount:         transaction.amount.to_s,
+            card_num:       card_num,
+            card_code:      '007',
             exp_date_month: expire_month,
-            exp_date_year: expire_year
+            exp_date_year:  expire_year,
+            currency:       transaction.currency_iso_code
```

Comment:
There's a lot of green and red, but the only addition (outside of specs) is this line.